android/clear_environment.sh

```diff
@@ -3,14 +3,20 @@ set -e
 
 # Script to clear environment BOINC for Android
 
-if [ ! -d ../../boinc/android ]] ; then
+if [ ! -d android ]] ; then
     echo You run this script from diffrent folder: $PWD
-    echo Please run it from boinc/android
+    echo Please run it from boinc
+    echo Type: \'android/clear_environment.sh\'
     exit 1
 fi
 
-rm -rf ../3rdParty/buildCache/
-rm -rf ../Makefile
-rm -rf ../m4/Makefile
+if [ "hard" = "$1" ]; then
```

Comment:
Instead of `hard` I'd suggest to use `full`

Reply:
I changed